firmware/lib/Comm/src/TestStandCommController.cxx

```diff
@@ -58,6 +50,24 @@ SerialResult TestStandCommController::position(int32_t x_counts, int32_t y_count
     return this->session.send_message(msg);
 }
 
+SerialResult TestStandCommController::temp(TempData *temp_data)
+{
+    TempMsgData data = {
+        .temp_ambient = htonl(round(temp_data->temp_ambient * temp_data_scaler)),
+        .temp_motor_x = htonl(round(temp_data->temp_motor_x * temp_data_scaler)),
+        .temp_motor_y = htonl(round(temp_data->temp_motor_y * temp_data_scaler)),
+        .temp_mpmt    = htonl(round(temp_data->temp_mpmt    * temp_data_scaler)),
+        .temp_optical = htonl(round(temp_data->temp_optical * temp_data_scaler))
```

Comment:
How many decimal places do we report for temperature data?

Reply:
Currently 6 (completely arbitrarily chosen), that's what the `1E6` multiplier is.
We read the number as a double, say `99.12345678`
Multiple by `1E6` and round: `99.12345678 * 1E6 = 99123456.78 --> 99123456`
Transfer the value as an integer: `99123456`
Divide by `1E6` on the host side: `99123456 / 1E6 = 99.123456`